### PR TITLE
Bump Mermaid-JS to major version 8

### LINKED ISSUE #2546: AEA-1621 Bump mermaid-js to latest version
**Is your feature request related to a problem? Please describe.**
The `mermaid-js` version used by the framework `7.1.2` is a bit obsolete. There is a major release `8` which adds useful features,
e.g. multiple `else` in an `alt` block.

However, it looks like there is a backward-incompatibility with the format at version `7`.

**Describe the solution you'd like**
Bump the version and fix all the broken diagrams.

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Bump Mermaid-JS to major version 8. In particular, to version `8.10.1`.

## Fixes

Fix #2546 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [C] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

n/a

### CODE REVIEW

File: docs/car-park-skills.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ This diagram shows the communication between the various entities as data is suc
         participant Car_Data_Buyer_AEA
         participant Car_Park_AEA
         participant Blockchain
-    
-        activate Car_Data_Buyer_AEA
+
         activate Search
+        activate Car_Data_Buyer_AEA
```
Comment (marcofavorito) [inline docs/car-park-skills.md:25]: just sorting

File: docs/car-park-skills.md [hunk 2]
```diff
@@ -38,8 +38,8 @@ This diagram shows the communication between the various entities as data is suc
         Car_Park_AEA->>Blockchain: check_transaction_status
         Car_Park_AEA->>Car_Data_Buyer_AEA: send_data
         
-        deactivate Client_AEA
         deactivate Search
+        deactivate Car_Data_Buyer_AEA
```
Comment (marcofavorito) [inline docs/car-park-skills.md:42]: missing

File: docs/erc1155-skills.md [hunk 3]
```diff
@@ -282,10 +282,10 @@ This diagram shows the communication between the various entities in this intera
         Deployer_AEA->>Blockchain: send_transaction
         Client_AEA->>ERC1155_contract: asks_balance
         
-        deactivate Deployer_AEA
         deactivate Search
+        deactivate Erc1155_contract
         deactivate Client_AEA
-        deactivate ERC1155_contract
```
Comment (marcofavorito) [inline docs/erc1155-skills.md:288]: note, here was capitalized, above is not

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**